ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/object-tracking.j2

```diff
@@ -0,0 +1,10 @@
+{# eos - object-tracking #}
+{% if track is arista.avd.defined %}
+!
+{%     for tracked_obj in track | arista.avd.natural_sort('name') %}
+{%         if tracked_obj.name is arista.avd.defined and tracked_obj.interface is arista.avd.defined %}
+{%             set tracked_obj_cli = "track " ~ tracked_obj.name ~ " interface " ~ tracked_obj.interface ~ " " ~ tracked_obj.tracked_property | arista.avd.default("line-protocol") %}
+{%         endif %}
+{{ tracked_obj_cli }}
```

Comment:
this needs to be inside the `if` to avoid errors in case the required variables are not defined.